src/main/java/uk/ac/ebi/eva/contigalias/controller/contigalias/ContigAliasController.java

```diff
@@ -300,12 +304,15 @@ private void linkPagedModelGetAssemblyByAuthority(
                     "nested inside it. The endpoint will either return a list of chromosomes or it will either return" +
                     " an HTTP error code 204 if no chromosomes are found or return an HTTP error code 400 if invalid " +
                     "parameters are found.")
-    @GetMapping(value = "chromosomes/{name}")
+    @GetMapping(value = "chromosomes/name/{name}")
     public ResponseEntity<PagedModel<EntityModel<ChromosomeEntity>>> getChromosomesByChromosomeNameAndAssemblyTaxidOrAccession(
-            @PathVariable @ApiParam(value = "Name of chromosome. Eg: HSCHR1_RANDOM_CTG5") String name,
+            @PathVariable @ApiParam(value = "Sequence name or UCSC style name of chromosome. Eg: HSCHR1_RANDOM_CTG5") String name,
             @RequestParam(required = false) @ApiParam(value = "Taxonomic ID of a group of accessions. Eg: 9606") Optional<Long> taxid,
             @RequestParam(required = false) @ApiParam(value = "Genbank or Refseq assembly accession. Eg: " +
                     "GCA_000001405.10") Optional<String> accession,
```

Comment:
I thought you renamed this to `asmAccession`. Did you do that in another branch? Just to be clear, in any endpoint that talks about both chromosomes and assemblies, if the assembly accession is mentioned, it should be `asmAccession` or similar, so that users know which accession is it.

Reply:
This is a good idea and I will do this once this at least this PR is merged so that I don't have to resolve variable name changes while rebasing the other branch to master.